capstone/src/main/webapp/youtubeTypescript.js

```diff
@@ -543,3 +544,50 @@ function formatCommentForSpreadsheet(comment) {
     formattedComment = formattedComment.replace(/\s+/g, " ");
     return formattedComment;
 }
+/** Creates chart of analyzed comments and requested attributes*/
+function drawTableChart() {
+    var requestedAttributes = getRequestedAttributes();
+    var tableData = new google.visualization.DataTable();
+    // Add columns
+    tableData.addColumn('string', 'COMMENT');
+    for (var i = 0; i < requestedAttributes.length; i++) {
+        tableData.addColumn('number', requestedAttributes[i]);
+    }
+    // Add rows
+    tableData.addRows(ANALYZED_COMMENTS.length);
+    for (var i = 0; i < ANALYZED_COMMENTS.length; i++) {
+        tableData.setCell(i, 0, ANALYZED_COMMENTS[i]);
+        for (var j = 1; j < ATTRIBUTE_DATA[i].length + 1; j++) {
+            tableData.setCell(i, j, ATTRIBUTE_DATA[i][j - 1]);
+        }
+    }
+    var table = new google.visualization.Table(document.getElementById('table-container'));
+    var formatter = new google.visualization.ColorFormat();
+    formatter.addRange(0, .2, 'black', '#F6F2FC');
+    formatter.addRange(.2, .4, 'black', '#E0CCFB');
+    formatter.addRange(.4, .6, 'black', '#A166F2');
+    formatter.addRange(.6, .8, 'white', '#8133EE');
+    formatter.addRange(.8, 1, 'white', '#6200EA');
+    for (var i = 0; i < requestedAttributes.length + 1; i++) {
+        formatter.format(tableData, i);
+    }
+    table.draw(tableData, { allowHtml: true, showRowNumber: false, width: '100%', height: '100%' });
+}
+/** Gives the appropriate color for a bar in a barchart given its score */
+function getColor(score) {
+    if (score >= 0.8) {
+        return '#6200EA'; // Darkest purple
+    }
+    else if (score >= 0.6) {
+        return '#8133EE'; // Dark purple
+    }
+    else if (score >= 0.4) {
+        return '#A166F2'; // Mild purple
+    }
+    else if (score >= 0.2) {
+        return '#E0CCFB'; // Light purple
+    }
+    else {
+        return '#F6F2FC'; // Lighest purple
+    }
+}
```

Comment:
Just FYI, this code exists in master branch's script.js (and has been updated)

Reply:
I've updated it in "update getColor"

Reply:
Unless something weird is happening in Typescript that I don't know about, I think it's possible to delete it entirely here to avoid code repetition

Reply:
I have added this file to .gitignore in "add gitignore and delete cache". This means that moving forward this file will not be checked in.